src/meta/src/hummock/compaction_group/manager.rs

```diff
@@ -108,15 +108,13 @@ impl<S: MetaStore> CompactionGroupManager<S> {
 
     /// Unregisters `table_fragments` from compaction groups
     pub async fn unregister_table_fragments(&self, table_fragments: &TableFragments) -> Result<()> {
```

Comment:
Get off topic, i think we don't need the whole `TableFragment` just using the `TableID`. For `register_table_fragments`, we just need to distinguish between `mview` and `internal state`. IIUC, we can fix it next PR.

Reply:
> LGTM for `creating_tables`, thanks !!!

I resolved notification on creating sources after #4817 merged. PTAL if I missed something. Hope ddl operation won't get stunk like before after this PR merged.